minvws/nl-kat-coordination
# Exclude Report from ooi list

### Changes

- Exclude Reports OOI from OOI list
- Remove Report from filter (filter by ooi type)

### Issue link

Closes ...

### Demo
You cannot filter by Report ooi type anymore, it is not listed for filters:
<img width="628" alt="image" src="https://github.com/user-attachments/assets/d6a85492-7bf3-45e4-bf29-4bd82447c72c">

Even though you filter it from the URL:
<img width="755" alt="image" src="https://github.com/user-attachments/assets/6ea8f4ef-94a0-4f4e-bea7-9c552b01cb10">

### QA notes

Go to ooi list and see that you cannot see "Report" ooi type in the filter and also Report OOIs are not listed.

---

### Code Checklist







---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
